plugins/foreman_openscap/0.6/index.md

```diff
@@ -150,6 +150,20 @@ Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate setti
 
 This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and triggering of foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
 
+Once installed, Foreman will automatically assign this class to hosts or host groups and populate the necessary parameters based on the settings you apply in Foreman under **COMPLIANCE**.
+
+To further ease the setup, the Foreman plugins yum repo can be set up if you define Foreman's major release version, see parameters for Yum repo below.
```

Comment:
nicely written, could you please somewhere mention that this is available since Foreman 1.15 or puppet module version 0.3.15 (both do not exist atm, but they will soon)